### PR TITLE
[Misc] Populated 20 sample person on address book

### PR DESCRIPTION
**Pull request by yewshengkai**
- [x] Populate 20 samples
- [x] "Organize the demo to present a cohesive picture of the product as a whole. Unlike in v1.2-v1.4 demos, there is no need for each person to demo their own features."

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -28,19 +28,61 @@
...
+                new Person(new Name("Rido Kuma"), new Phone("91235874"), new Email("ridokuma@example.com"),
+                        new Address("Blk 51 Indian Street 09, #21-11"), new Remark("my best friend"),
+                        new Avatar(""), getTagSet("friends")),
+                new Person(new Name("Seri Lanka"), new Phone("91235874"), new Email("serilanka@example.com"),
+                        new Address("Blk 01 NotSingapore Street 01, #01-01"), new Remark("doesn't stay in malaysia"),
+                        new Avatar(""), getTagSet("classmates")),
                 new Person(new Name("Roy Balakrishnan"), new Phone("92624417"), new Email("royb@example.com"),
                     new Address("Blk 45 Aljunied Street 85, #11-31"), new Remark(""),
-                    new Avatar(""), getTagSet("colleagues"))
+                    new Avatar(""), getTagSet("colleagues")),
```
Comment (karrui) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java]: The comma seems unneeded

Comment (yewshengkai) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java]: yep. done. Thanks X)

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 2]
```diff
@@ -28,16 +28,58 @@
                     getTagSet("friends")),
                 new Person(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
                     new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"), new Remark("family of mouse"),
-                    new Avatar(""), getTagSet("colleagues", "friends")),
+                    new Avatar("https://i.imgur.com/xPHOeWL.png"), getTagSet("colleagues", "friends")),
```
Comment (karrui) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java]: remove this line, don't risk app crashing when no internet connection to convert

**Comment by yewshengkai** [comment]
PR closed. SampleData.xml has an updated version.
Reopen only if this is needed and updated accordingly.

**Pull request closed without merging**